### PR TITLE
Re-write `DraftEditorExample` in the website using react hooks

### PR DESCRIPTION
**Pull request by mrkev**
Also took the oportunity to clean up the component- it had unused imports and `_handleKeyCommand` was returning a boolean instead of `"handled"` and `"not-handled"`.

`yarn && yarn start`, used the editor.

### CODE REVIEW

**Comment by claudiopro** [comment]
Good job! What about integrating the repo with something like Zeit to deploy the PR for easier testing?

Hey @yangshun how did you do that for your Docusaurus 2 PR, did you deploy manually to now.sh?

File: website/src/components/DraftEditorExample/index.js [hunk 1]
```diff
@@ -6,111 +6,102 @@
...
-    return getDefaultKeyBinding(e);
-  }
+function RichEditorExample(props) {
+  const [editorState, setEditorState] = useState(EditorState.createEmpty());
+  const editor = useRef(null);
 
-  _toggleBlockType(blockType) {
-    this.onChange(RichUtils.toggleBlockType(this.state.editorState, blockType));
-  }
+  const focus = () => {
```
Comment (claudiopro) [inline website/src/components/DraftEditorExample/index.js]: Nit: can you rename to `focusEditor`? This is shadowing the `window.focus()` method

**Comment by yangshun** [comment]
> Hey @yangshun how did you do that for your Docusaurus 2 PR, did you deploy manually to now.sh?

We use Netlify previews. However, Netlify is recently tighten its free tier limits so it might not be that viable as an option. You could check out Netlify or Now.

**Comment by claudiopro** [comment]
@mrkev I don't want to block on this, let's land as-is and fix forward

**Pull request closed without merging**